scala-package/pom.xml

```diff
@@ -427,7 +433,12 @@
     <dependency>
       <groupId>org.scala-lang.modules</groupId>
       <artifactId>scala-parser-combinators_2.11</artifactId>
-      <version>1.0.4</version>
+      <version>1.0.5</version>
+    </dependency>
+    <dependency>
+      <groupId>org.scala-lang.modules</groupId>
+      <artifactId>scala-xml_2.11</artifactId>
```

Comment:
This dependency wasn't there before right ? 
Any reason to add it ?

Reply:
It got injected automatically, but with the wrong version of scala. I added it manually to use the correct version